modules/gateways/blockonomics/assets/templates/no_crypto_selected.tpl

```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" type="text/css" href="{$WEB_ROOT}/modules/gateways/blockonomics/assets/css/order.css">
+
+<div class="bnomics-order-container">
+    <h3>
+        No crypto currencies are enabled for checkout
```

Comment:
We do not [validate the currencies in woocommerce](https://github.com/blockonomics/woocommerce-plugin/blob/master/blockonomics-woocommerce.php#L244-L254). Lets remove the currency validation from WHMCS as we now have the no_crypto_selected template as with Woocommerce: https://github.com/blockonomics/whmcs-bitcoin-plugin/blob/master/modules/gateways/blockonomics.php#L358-L374